Gemfile

```diff
@@ -3,25 +3,22 @@
 source 'https://rubygems.org'
 
 gem 'rake', require: false
-# SDK feature dependencies
+
 gem 'aws-crt' if ENV['CRT']
 gem 'base64'
 gem 'bigdecimal'
 gem 'csv'
 gem 'http-2'
+gem 'irb'
 gem 'jmespath'
-if defined?(JRUBY_VERSION)
-  # get the latest jruby-openssl to support sigv4a
-  # see: https://github.com/jruby/jruby-openssl/issues/30
-  gem 'jruby-openssl'
-end
+gem 'jruby-openssl' if defined?(JRUBY_VERSION)
+gem 'rdoc'
```

Comment:
for context purposes: where do we use rdoc gem?

Reply:
I'm not sure - we got the warning. Maybe from yard.